scripts/train_lm.py

```diff
@@ -63,6 +68,8 @@
                     help="Directory containing the training text.")
 parser.add_argument("order",
                     help="Order of N-gram model to be trained.")
+parser.add_argument("work_dir",
+                    help="Working directory for building the language model.")
 parser.add_argument("lm_dir",
```

Comment:
Maybe just to make sure that it works with the default, yes.  I'll test it
with the provided args.



On Mon, Sep 5, 2016 at 2:12 AM, Wang Jian <notifications@github.com> wrote:

> In scripts/train_lm.py
> <https://github.com/danpovey/pocolm/pull/66#discussion_r77473918>:
>
> > @@ -63,6 +68,8 @@
> >                      help="Directory containing the training text.")
> >  parser.add_argument("order",
> >                      help="Order of N-gram model to be trained.")
> > +parser.add_argument("work_dir",
> > +                    help="Working directory for building the language model.")
> >  parser.add_argument("lm_dir",
>
> Ok. And do i need to revert the changes in run.sh to used the default
> value in poco egs?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/danpovey/pocolm/pull/66/files/dfbfb3a72385f08ee4738dfe8aba8d5656ee6c07#r77473918>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADJVu7h69Oewlu47dZ2DNkhnc_WM0taYks5qm7LqgaJpZM4J0wVP>
> .
>